wgsl/index.bs

```diff
@@ -6699,30 +6712,43 @@ An `if` statement is executed as follows:
     | [=syntax/expression=]
 </div>
 
-A <dfn noexport dfn-for="statement">switch</dfn> statement transfers control to one of a set of case clauses, or to the `default` clause,
-depending on the evaluation of a selector expression.
+A <dfn noexport>case clause</dfn> is the [=syntax/case=] token followed by a list of [=syntax/case_selectors=] and a
+<dfn noexport>case body</dfn> in the form of a [=compound statement=].
 
-If the selector value equals a value in a case selector list, then control is transferred to
-the body of that case clause.
-The expressions in the [=syntax/case_selectors=] [=shader-creation error|must=]
-be [=const-expressions=].
-If the selector value does not equal any of the case selector values, then control is
-transferred to the `default` clause.
+A <dfn noexport>default-alone clause</dfn> is the [=syntax/default=] token followed by a <dfn noexport>default-alone body</dfn>
+in the form of a [=compound statement=].
+
+A <dfn noexport>default clause</dfn> is either:
+* a [=case clause=] where [=syntax/default=] appears as one of its selectors, or
+* a [=default-alone clause=].
+
+Each switch statement [=shader-creation error|must=] have exactly one [=default clause=].
 
-Each switch statement [=shader-creation error|must=] have exactly one default clause.
+The [=syntax/default=] token [=shader-creation error|must=] not appear more than once in a single [=syntax/case_selector=] list.
 
 [=Type rule precondition=]:
 For a single switch statement, the selector expression and all case selector expressions [=shader-creation error|must=] be of the same [=type/concrete=] [=integer scalar=] type.
 
-A literal value [=shader-creation error|must not=] appear more than once in the case selectors for a switch statement.
+The expressions in the [=syntax/case_selectors=] [=shader-creation error|must=]
+be [=const-expressions=].
+
+Two different case selector expressions in the same switch statement [=shader-creation error|must not=] have the same value.
 
-Note: The value of the literal is what matters, not the spelling.
-For example `0` and `0x0000` both denote the zero value.
+If the selector value equals the value of an expression in a [=syntax/case_selector=] list,
+then control is transferred to the body of that [=case clause=].
+If the selector value does not equal any of the case selector values, then control is
+transferred to the body of the [=default clause=].
 
-When control reaches the end of a case body, control normally transfers to the first statement
+When control reaches the end of a [=case body=] or [=default-alone body=], control transfers to the first statement
```

Comment:
This strikes me as a bit unfortunate. Can we earlier say the case clause and default-alone clause both have a compound statement? And them, this compound statement is referred to as the case body.

Having two different body types seems unnecessary.

Reply:
Addressed by removing the definitions of the body types.  Just wrote "body of ..."